paasta_tools/cli/utils.py

```diff
@@ -1083,3 +1101,31 @@ def get_paasta_oapi_api_clustername(cluster: str, is_eks: bool) -> str:
     "eks-" prefix
     """
     return f"eks-{cluster}" if is_eks else cluster
+
+
+def get_current_ecosystem() -> str:
+    """Infer ecosystem from local-run configuration"""
+    local_run_config = load_system_paasta_config().get_local_run_config()
+    ecosystem = local_run_config["default_cluster"].split("-", 1)[-1]
+    return f"{ecosystem}prod" if ecosystem == "corp" else ecosystem
```

Comment:
It's the former. The fact is that this codebase does not make references to our internal filesystem structure in that manner, so I wanted to keep it that way.

Reply:
gotcha - imo, referencing `/nail/etc` would probably be preferable since folks wouldn't be aware that `default_cluster` value in SystemPaastaConfig is being used for locating where a box is hosted without reading this code 

e.g., if we were to force that users specify a --cluster arg in local-run and cleaned up the SystemPaastaConfig in puppet, we'd be in for a bit of a surprise once this stopped working

(and fwiw, we're already reading /nail/etc/runtimeenv in this file (and other /nail or /nail/etc/ filelocs in other files))

Reply:
if we really wanted to make this flexible, we could maybe add a little bit more  SystemPaastaConfig to make some of these dirs configurable - but a lot of the concepts are pretty yelp-centric anyway (and we're really more developing in the open than anything else here :p) so it doesn't seem like we'd gain much embarking on that effort